src/components/transaction-graphic/MosaicAddressRestrictionGraphic.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import GraphicComponent from './graphics/GraphicComponent.js';
+import AccountIcon from './graphics/AccountIcon.js';
+import CircleRestriction from './graphics/CircleRestriction.js';
+import Arrow from './graphics/Arrow.js';
+import Svg, { Text } from 'react-native-svg';
+
+export default class MosaicAddressRestrictionGraphic extends GraphicComponent {
+    get circleIconsToDisplay() {
+        return [true];
+    }
+
+    render() {
+        return (
+            <Svg
+                x={0}
+                y={0}
+                width={this.transactionGraphicWidth}
+                height={this.transactionGraphicHeight}
+                viewBox={this.transactionGraphicViewbox}
+                style={this.styles.transactionGraphicSvg}
+            >
+                <AccountIcon
+                    x={this.subjectPositionX}
+                    y={this.subjectPositionY}
+                    width={this.subjectWidth}
+                    height={this.subjectHeight}
+                    address={this.props.signerAddress}
+                />
+                <AccountIcon
+                    x={this.objectPositionX}
+                    y={this.objectPositionY}
+                    width={this.subjectWidth}
+                    height={this.subjectHeight}
+                    address={this.props.targetAddress}
+                />
```

Comment:
In explorer we are using `MosaicIcon` not sure are you intend to use it.

Reply:
Yes, I changed that because the target is an address, not a mosaic.

Reply:
Alright, it should do the same in Explorer as well 👍🏼

Reply:
Agree, let's revisit it when we have the new design ready